db/migrate/20140716131107_create_category_translation_tables.rb

```diff
@@ -112,16 +112,16 @@ def down
     new_categories = []
     PublicBodyCategory.all.each do |category|
       category.locale = category.translation.locale.to_s
-      FastGettext.default_available_locales.each do |locale|
-        if locale.to_s != category.locale
+      AlaveteliLocalization.available_locales.each do |locale|
+        if locale != category.locale
```

Comment:
ok, at that point we've just reinstated the `locale` database column so `category.locale` is a `String` drawn from the database, not a translated attribute (until we do that, there is no `.locale` method)